### DISCUSSION TITLE
Please tell us about your use cases for sast-scan

### DISCUSSION THREAD

**Discussion by prabhu** [ShiftLeftSecurity/sast-scan#381] [Q&A]
I am seeing the usage of scan (in terms of docker pulls) growing organically.

![D8147891-E677-41EC-B1EA-0AF28AE00A70](https://user-images.githubusercontent.com/7842/216126733-5b980d7c-67d7-44cd-a2b9-f45d7e7428be.jpeg)

This is despite not adding a single new feature since [2021](https://github.com/ShiftLeftSecurity/sast-scan/issues/352). I am keen to hear the use cases this project is solving/not solving for you and your organization.

You can reach me on prabhu at appthreat.com or on [discord](https://discord.gg/DCNxzaeUpd)

**Comment by mkadp** [comment]
Hi Prabu,

Our reason for starting to use slscan was that we are going for ISO27001 compliance and are using a platform called Sprinto to manage the auditing and compliance process. This tool has automatic integration with SLscan. We compared it to Snyk which is also supported but requires an Enterprise plan which is too expensive for our small team.

The initial integration was rather quick and easy to set up. But ongoing I have seen a number or problems with it particularly around the inclusion of dev dependencies in the vulnerability scan and a massive increase of reported vulnerabilities on projects that haven't had any changes in the last few months.

cheers,
Mk

**Reply by prabhu** [reply]
@mkadp Thank you for your support and patience. This issue is resolved with the latest scan image and version 2.1.2. Please retest and let me know how it looks.
